temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using hardcoded strings directly in the code. Consider defining them as constants or using a configuration file.
```
Define a constant for the string "hello" and use that constant in the print statement.
```